figs/stanplots/analyses_wanang.R

```diff
@@ -1,31 +1,43 @@
-# Specify a path for data and rds files
-setwd("C:\\Users\\Piotr Szefer\\Desktop\\Work\\garden experiment\\figs\\stanplots")
+
 
 # Examples
 # https://www.sciencedirect.com/science/article/pii/S0095447017302310
 
-# Upload data
-AllTestData <- read.table("all_test_data.txt")
-TreeTestDataset <- read.table("tree_test_data.txt")
+
+# load data and packages --------------------------------------------------
+
+# load data
+AllTestData <- read.table("figs/stanplots/all_test_data.txt")
```

Comment:
it isn't useful to use setwd() when you have Rstudio projects! now the working directory is automatically set to where the .Rproj file lives (at the highest level of the directory) and we can specify a relative path from there